src/debug.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "debug.h"
 
+#define MAX_ATTR_SIZE 512
```

Comment:
It is a [good practice](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es31-dont-use-macros-for-constants-or-functions) to not use macros for constants.
In this case `static const size_t maxAttrSize = 512;` would be more appropriate.

Reply:
Thank you. 
Updated the PR.